azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -214,6 +215,7 @@ internal open class CallCompositeActivity : AppCompatActivity() {
 
             if (isFinishing && store.getCurrentState().navigationState.navigationState == NavigationStatus.EXIT) {
                 store.dispatch(CallingAction.CallEndRequested())
+                callDurationManager.onReset()
```

Comment:
We can not bind any call logic to the activity lifecycle.
It needs to be done in the Middleware based on the call status change or on `CallEndRequested`.